keps/sig-apps/2214-indexed-job/README.md

```diff
@@ -459,8 +463,7 @@ gate enabled and disabled.
 
 #### Beta -> GA Graduation
 
-- E2E test graduates to conformance
-- 2 examples of operators using Indexed Jobs.
+- [E2E](https://testgrid.k8s.io/sig-apps#gce&include-filter-by-regex=Indexed) test graduates to conformance
```

Comment:
Here, similarly to ready pods, I'm curious about scalability tests, both from the job size increase pov and from a job controller increased processing time, how big impact running index job is, basically is the answer I'm looking for.

Reply:
I don't think there is any significant difference from a regular parallel Job. Are there any precedents of scalability tests for those that I can build on?

Reply:
Added as graduation criteria